### PR TITLE
[MODULAR] Resprited human body parts

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Because a good portion of the community dislikes the new human sprites, it was decided to temporarily revert them, until something was done. It was roughly a month ago, and since then no one has done anything.

Because it's kinda hard to make decent sprites around these resprited bodyparts if the server's still using the old one, I've decided to deal with this personally by respriting these sprites based on the criticism from both tg and skyrat, which was mainly focused about portions of the outlines and inlines being too dark, thus giving the sprite a rougher, sharper appearance, well-expressed by the reaction images posted in the "humanening" PR.

Another minor grudge, which is also why I'm not really PR'ing this upstream, is the extra pixel under the male torso sprite (jokingly called "dixel") and the vague resemblance of a bosom on the sides of the female chest sprites, which have been removed since overlays for those exist already.

However, because the human head, chest, arms and legs in the code are basically the master type of every other "mutant" part, modularisation is not a viable option, so, until a way is figured out, this will be a direct edit of the main human_parts_greyscale.dmi file. EDIT: THAT'S NO LONGER THE CASE.

## How This Contributes To The Skyrat Roleplay Experience
OLDER SPRITES (credits to Simplehorror for the image in the original humanening PR):

OLD SPRITES (keep in mind extra pixels to the female back were added later, creds to axietheaxolotl):

NEW SPRITES:
neo

Nothing has changed drastically, just a little "smoother" now (the different bg and skin tone of the screenhots might make comparison a little harder).

## Changelog

:cl:
imageadd: resprited human body parts. They should look a tad smoother and less "overshaded" now.
/:cl:

### CODE REVIEW

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Is there any reason why we can't make this modular?

**Comment by GoldenAlpharex** [comment]
There's also some extra pixels on one of the legs when aiming west.
image

**Comment by Ghommie** [comment]
Ok, I've fixed the issue with the flipped leg and made the changes modular as requested.

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
This style of modularisation is not allowed, please follow the guide by making a module, one already exists, add it to that.

**Comment by Ghommie** [comment]
> This style of modularisation is not allowed, please follow the guide by making a module, if one already exists, add it to that.

Isn't that what I'm doing already? I could swear the code is all found in the `modular_skyrat/master_files/code/modules/surgery/bodyparts/species_parts` folder while the icons are in the `modular_skyrat/master_files/icons/mob/species` one.

EDIT: Both are pre-existing folders.

**Comment by tf-4** [comment]
We're trying to avoid using master files where possible is what cobalt means

**Comment by Acer202** [comment]
You made these look 1000 times better. Don't let this die!

**Comment by tf-4** [comment]
You need to address reviews @Ghommie , I can't keep the stalebot at bay forever.

**Comment by Ghommie** [comment]
Sorry, I've been doing other things these days. I haven't forgotten about this.

**Comment by Ghommie** [comment]
Ok, I've moved the involved files from `master_files` to a new module in the `modules` folder as requested, complete of code and icons subfolders.

File: modular_skyrat/modules/bodyparts/code/skrell_bodyparts.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Skrell!
+/obj/item/bodypart/head/mutant/skrell
+	icon_greyscale = 'modular_skyrat/modules/bodyparts/icons/skrell_parts_greyscale.dmi'
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/bodyparts/code/skrell_bodyparts.dm]: I feel like we should probably consider making all of these paths defines, even if they're just defined for that file alone, just so it's easier to maintain in the future if they're ever moved around again.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I'd just be curious to see what you think about this one.

**Comment by desertmote** [comment]
well done!

File: modular_skyrat/modules/bodyparts/code/skrell_bodyparts.dm (hunk 1 above)
Comment (Ghommie) [reply modular_skyrat/modules/bodyparts/code/skrell_bodyparts.dm]: After some considerations, while I'm not enthusiast of the idea - as you said, most of these defines are only used in one file -, it doesn't hurt to try.

**Review by GoldenAlpharex — APPROVED** [review]
Very nice!

**Review by tf-4 — APPROVED** [review]

**Comment by GoldenAlpharex** [comment]
Let's get that merge!

**Pull request merged**